### DISCUSSION TITLE
Can not upgrade from 1.6.2 to 1.8 Error: Chia can not be closed

### DISCUSSION THREAD

**Discussion by malahhaor** [Chia-Network/chia-blockchain#15239] [Support]
Running 1.8 installer. Half way through install receiving an error that states; Chia can not be closed. Please close it manually and click retry.
The client is NOT running. System has been rebooted. Installing with Admin, nothing works.
This is the first time i've ever had this issue. Others reported similar in the past but no one provided a solution. 
Can anyone suggest what services maybe hanging that i need to stop manually?

**Comment by malahhaor** [comment]
Anyone? Support?

**Reply by rushil-patel** [reply]
I'm facing the same error

**Comment by iTZUNAMI** [comment]
windows or linux?

**Reply by rushil-patel** [reply]
Windows. My resolution was to uninstall and then reinstall the app.

**Reply by iZakaroN** [reply]
I have exactly the same problem with 1.8.0 and now with 1.8.1. Restarting the PC does not help. Going to try to uninstall first.

**Comment by iZakaroN** [comment]
I have exactly the same problem with 1.8.0 and now with 1.8.1. Restarting the PC does not help, but uninstalling previous version first solve the problem.

**Comment by bitsam** [comment]
Any tips on what to be careful of when un-install / re-install? I.e. did you have to use your seed words? Does it keep the chain and wallet intact?

**Reply by iTZUNAMI** [reply]
you can uninstall/install without any worrys.
If your mainet db is already synced you can copy and paste later after a new installation. Should be in C:\Users\yourusername\.chia\mainnet\db\blockchain_v2_mainnet.sqlite like 100GB.

If you reinstall yes you have to restore the seed from the 24 unique words of your wallet.
Also you can download a torrent from chia website under the download tab with the DB updated to last month, so you don't have to resync with the chia client that is slow, just copy paste the DB and sync last month in few hours.

**Reply by bitsam** [reply]
Thank you. Unfortunately, I lost the DB when I uninstalled the old version. I think I had the DB nested in the same folder as the application. Thanks for the advice on torrent for the DB, as it was taking forever. Also, I was able to start the chia daemon without needing the seed words to restore my wallet. Thanks, @iTZUNAMI again for the quick response.
